subhalo_properties.py

```diff
@@ -136,6 +136,18 @@ def pos_star(self):
     def pos_baryons(self):
         return self.position[self.baryons_mask_sh]
 
+    @lazy_property
+    def max_pos(self):
+        return self.position.max(
+            axis=0
+        ) * 1.001 + self.centre
+
+    @lazy_property
+    def min_pos(self):
+        return self.position.min(
+            axis=0
+        ) * 1.001 + self.centre
```

Comment:
This will generate positions outside the box sometimes, since all of the particles in the position array have been wrapped to the periodic copy nearest the centre. Maybe that's ok. It guarantees that pos_max > pos_min, at least!

Reply:
I don't have a strong opinion on whether we should be wrapping the output positions, I think either way could confuse end users. I implemented it like this since swiftsimio's spatial masking can handle values outside the box, but it doesn't like pos_max < pos_min.

Reply:
I'll leave it this way unless you have a preference for wrapping. I'll add a comment to the property description saying that the values can be outside the box.